__tests__/route-matching-general-tests.ts

```diff
@@ -295,4 +295,23 @@ describe('Route Matching - willReturn', () => {
         expect(fakeServer.hasMade(route.call.withPath(actualPath).withBodyText(actualBody))).toEqual(true);
         expect(fakeServer.hasMade(route.call.withBodyText(actualBody).withPath(actualPath))).toEqual(true);
     });
+
```

Comment:
Add some tests for multiple restrictions now that it's supported?

Reply:
It's not supported